### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by AdiGajbhiye**
## Overview

- hide `Sync with DB` button for ephemeral
- flashing tooltip in select/non-select info button in cll
- disallow column selection for ephemeral
- better error message around credentials

## Checklist

- [ ] I have run this code and it appears to resolve the stated issue
- [ ] `README.md` updated and added information about my change

### CODE REVIEW

**Review by mdesmet — APPROVED** [review]

File: src/altimate.ts [hunk 1]
```diff
@@ -207,7 +207,11 @@ export class AltimateRequest {
           "Content-Type": "application/json",
         },
       });
-      if (response.status === 401 || response.status === 403) {
```
Comment (anandgupta42) [inline src/altimate.ts]: @AdiGajbhiye Can you add a telemetry event also?

File: src/altimate.ts [hunk 2]
```diff
@@ -200,6 +207,13 @@ export class AltimateRequest {
           "Content-Type": "application/json",
         },
       });
+      if (
+        response.status === 401 ||
+        response.status === 403 ||
+        response.status === 404
+      ) {
+        window.showErrorMessage("Invalid credentials");
```
Comment (mdesmet) [inline src/altimate.ts:215]: I think this error message can be improved. What should they do?

File: src/altimate.ts [hunk 3]
```diff
@@ -187,6 +187,13 @@ export class AltimateRequest {
       return;
     }
 
+    if (!config.instance || !config.key) {
+      window.showErrorMessage(
+        "Credentials are not set properly. Please refer to Altimate docs.",
```
Comment (mdesmet) [inline src/altimate.ts:192]: Add link to docs.

**Review by mdesmet — COMMENTED** [review]
LGTM except copy.

**Review by mdesmet — APPROVED** [review]

**Pull request merged**